packages/python-jinja2/python-jinja2.spec

```diff
@@ -26,7 +26,7 @@ BuildRequires:  %{?scl_prefix}python%{python3_pkgversion}-setuptools
 %package -n     %{?scl_prefix}python%{python3_pkgversion}-%{srcname}
 Summary:        %{summary}
 %{?python_provide:%python_provide python%{python3_pkgversion}-%{srcname}}
-Requires:       %{?scl_prefix}python%{python3_pkgversion}-markupsafe >= 0.23
+Requires:       %{?scl_prefix}python%{python3_pkgversion}-MarkupSafe >= 0.23
```

Comment:
the RPM doesn't provide that name, so you either need to lowercase it again, or fix the markupsafe rpm to add that provides